cmd/eskip/eskip.go

```diff
@@ -113,6 +113,18 @@ func main() {
 		exitHint(err)
 	}
 
+	in, out, err = addDefaultMedia(cmd, in, out)
+
+	if err != nil {
+		exitHint(err)
+	}
+
+	writeClient, err := createWriteClient(out)
```

Comment:
maybe this createWriteClient could go inside the command, only when it is needed, similarly to read client?